rust/oak/src/lib.rs

```diff
@@ -66,6 +68,23 @@ impl Write for Channel {
     }
 }
 
+/// Read a serialized protobuf request message from a Channel.
+pub fn read_proto_message<T: ::protobuf::Message>(grpc_channel: &mut Channel) -> T {
```

Comment:
I was trying to avoid adding a dependency from `oak` to `protobuf`, as Oak should be agnostic of whatever channels pass around, and ideally only talk about bytes. But then again it is convenient to have these methods here... Maybe we should have an `oak_grpc` crate though that encapsulates this stuff, and ideally that will also help with #11. WDYT?

Reply:
That makes sense &ndash; the base Oak layer deals with bytes, but have a higher-level helper library that's available for those Oak Modules that work in gRPC-land.  I'll rearrange things and rebase back to scratch.